docs/getting-started/install-configure.md

```diff
@@ -447,12 +468,41 @@ spec:
       name: azure-creds
       key: creds
 ```
+
 ```console
 kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/master/docs/snippets/configure/azure/providerconfig.yaml
 ```
 
 </div>
+<!-- End Azure Tab -->
+
+</div>
+<!-- End Cloud Provider Tabs -->
+
+</div>
+<!-- End Upstream Tab -->
+
+<!-- Begin Downstream Tab -->
+<div class="tab-pane fade" id="using-a-downstream-distro" markdown="1">
+
+## Start with a Downstream Distribution
```

Comment:
Is the intention for the Crossplane getting started guide to work with downstream distributions as well (as they should be compatible)? If so, should this section come before installing configurations?

Reply:
What I'm proposing here is to route users to the docs site for whatever downstream distribution they want to use vs. put getting started documentation for that distro in the Crossplane documentation site itself. So the path would be "use upstream and continue reading, or choose a distro and view the distro's quickstart". It feels like today there's a lot of jumping back and forth between Upbound docs and Crossplane docs with the way things are structured now.

Reply:
@grantgumina that makes a lot of sense 👍🏻 I think conformant "downstream" distributions should all be able to support the Crossplane getting started guide as well, but it is not _technically_ required that this getting started guide "works" (depending on how you define that) in every downstream distribution, so it makes sense to defer the the downstream docs. In practice we could advise any folks who want to add their downstream Crossplane distros to this section to make them work with the Crossplane getting started guide based on Upbound's experience of users getting confused if it does not work 👍🏻

Reply:
> In practice we could advise any folks who want to add their downstream Crossplane distros to this section to make them work with the Crossplane getting started guide based on Upbound's experience of users getting confused if it does not work

I like that